readme.md

```diff
@@ -12,22 +13,22 @@
 - [Icon Fonts](#icon-fonts)
 - [Stock Photos](#stock-photos)
 - [Stock Videos](#stock-videos)
-- [Stock Music & Sound Effects](#stock-music--sound-effects)
-- [Vectors & Clip Art](#vectors--clip-art)
-- [Product & Image Mockups](#product--image-mockups)
-- [HTML & CSS Templates](#html--css-templates)
+- [Stock Music \& Sound Effects](#stock-music--sound-effects)
+- [Vectors \& Clip Art](#vectors--clip-art)
+- [Product \& Image Mockups](#product--image-mockups)
+- [HTML \& CSS Templates](#html--css-templates)
```

Comment:
Same comment as before, Please revert the esc slash `\&`

Reply:
sure .. i've reverted the esc slash \&